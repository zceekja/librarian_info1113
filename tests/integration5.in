add collection book1.csv
add collection book2.csv
add member Tony Stark
add member Peter Parkrt
add member T'Challa
add member Steve Rogers
rent 100000 111111
rent 100000 111111
rent 100000 111112
rent 100000 111113
rent 100000 111114
rent 100000 111115
rent 100000 111116
relinquish 100000 111116
relinquish 100000 111115
relinquish 100000 111114
relinquish 100000 111113
relinquish 100000 111112
relinquish 100000 111111
rent 100001 111111
rent 100001 111113
rent 100001 111114
rent 100001 111115
rent 100001 111116
relinquish all 100001
rent 100002 111111
rent 100002 111112
rent 100002 111114
rent 100002 111115
rent 100002 111116
relinquish all 100002
rent 100003 111111
rent 100003 111112
rent 100003 111113
rent 100003 111115
rent 100003 111116
relinquish all 100003
common 100000 100001 100002 100003
list genres 
relinquish 100000 111111
exit