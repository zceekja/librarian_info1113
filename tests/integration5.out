user: 17 books successfully added.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
War

user: Unable to return book.

user: Ending Library process.
